.github/workflows/deploy.yml

```diff
@@ -55,4 +58,17 @@ jobs:
           environment: ${{ inputs.environment }}
 
       - name: Deploy release
-        run: make release-deploy APP_NAME=${{ inputs.app_name }} ENVIRONMENT=${{ inputs.environment }} IMAGE_TAG=${{ needs.database-migrations.outputs.commit_hash }}
+        id: deploy-release
+        run: |
+          make release-deploy APP_NAME=${{ inputs.app_name }} ENVIRONMENT=${{ inputs.environment }} IMAGE_TAG=${{ needs.database-migrations.outputs.commit_hash }}
```

Comment:
deploy release
deploy release
release deploy 

@_@

(not worth changing it)

Reply:
That's intentional. The style guide for step names is imperative tone, e.g. "Checkout code", "Build image", "Deploy release" etc.

The make commands are organized in a hierarchical fashion like a directory structure so that the commands can be grouped together e.g. `make infra-*` for infra commands, `make e2e-*` for e2e testing commands, `make release-*` for release management commands.